package.json

```
@@ -16,6 +16,10 @@
   },
   "devDependencies": {
     "@ionic/core": "^5.3.2",
-    "@stencil/core": "^2.0.1"
+    "@stencil/core": "^2.0.1",
+    "@types/jest": "^26.0.24",
+    "jest": "^26.6.3",
```

Ah, it looks like we need to add an explicit dependency on `jest-cli` too - I thought that wouldn't be needed as it's a subpackage of `jest`, my mistake! Would you mind adding one more dependency?
```suggestion
    "jest": "^26.6.3",
    "jest-cli": "^26.6.3",
```

---

Hm, I thought the same as well. Thanks for the suggestion!